server.js

```diff
@@ -140,6 +142,32 @@ async function lookupRecentQuizzes(count, completion) {
 
 ///SECTION: DB utility functions
 
+// Backs up database to a JSON file with name backup<date>.json.
+// ⚠️ Don't run in nodemon or it will cycle forever!
+function backupDatabaseToJSON() {
+    console.log("💾  Backing up database...");
+
+    database.collection(quizCollection).find({}).toArray(function(err, allQuizzes) {
```

Comment:
Can't run the server I'm getting this error:

TypeError: Cannot read property 'collection' of undefined

this line

Reply:
@MichaelJereza Could you tell me what steps you took that caused this issue? Sounds like the database didn't get set up properly, but I can't think why... 🧐

Reply:
i added an if statement that can be set to `true` to run the backup. Maybe give it a try now?